### DISCUSSION TITLE
Account Flagged Without Reason – Seeking Community Assistance

### DISCUSSION THREAD

**Discussion by ghost** [community/community#140033] [Discussions]
Hello GitHub Community,

I am reaching out for assistance regarding my GitHub account, which has been flagged without any clear reason. I have followed all community guidelines and have only used my account to showcase personal projects.

Recently, I created a new account to continue my work, but unfortunately, that account was also flagged shortly after. I have attempted to contact GitHub Support multiple times, but I have not received a response.

This situation is significantly impacting my ability to manage and share my projects. If anyone has experienced a similar issue or has any insights on how to resolve this, I would greatly appreciate your help.

Thank you for your time!

Best,
[Ömer Çıkan]

**Comment by LiteBrite82** [comment]
Hi @omercikann  

Is the account issue specific to your access to Community Discussions? Or have your accounts been closed entirely?
